File: C/Grace/Grace.c
#include <stdio.h>
#include <fcntl.h>
#include <unistd.h>
#define PUT(x) int main(){int fd = open("Grace_kid.c", 0100 | 01 | 01000, 0644);if (fd == -1) return (1);dprintf(fd, "#include <stdio.h>\n#include <fcntl.h>\n#include <unistd.h>\n#define PUT(x) "x"\n#define STR(s) STR_HELPER(s)\n#define STR_HELPER(s) #s\n/*\n\tIt is not in the stars to hold our destiny but in ourselves.\n*/\nPUT(STR(PUT(x)))");if (close(fd) == (-1)) return (1);return(0);}
#define STR(s) STR_HELPER(s)
#define STR_HELPER(s) #s
/*
	It is not in the stars to hold our destiny but in ourselves.
*/
PUT(STR(PUT(x)))

File: RUST/Grace/Grace.rs
/*
	It is not in the stars to hold our destiny but in ourselves.
*/
use std::fs::File;
use std::io::Write;
macro_rules! mainer {
	() => {
		fn main()
		{
			let s1 = "/*\n\tIt is not in the stars to hold our destiny but in ourselves.\n*/\nuse std::fs::File;\nuse std::io::Write;\nmacro_rules! mainer {\n\t() => {\n\t\tfn main()\n\t\t{\n\t\t\tlet s1 = ";
			let s2 = "let mut file = File::create(\"Grace_kid.rs\").expect(\"Unable to create file\");\n\t\t\twrite!(file, \"{}{:?};\n\t\t\tlet s2 = {:?};\n\t\t\t{}\", s1, s1, s2, s2).expect(\"Unable to write file\");\n\t\t\tdrop(file);\n\t\t}\n\t};\n}\nmainer!();";
			let mut file = File::create("Grace_kid.rs").expect("Unable to create file");
			write!(file, "{}{:?};
			let s2 = {:?};
			{}", s1, s1, s2, s2).expect("Unable to write file");
			drop(file);
		}
	};
}
mainer!();